.changeset/slimy-taxis-live.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/plugin-search-backend-node': patch
```

Comment:
I don't know how many people are using the `setTranslator()` method on `LunrSearchEngine` at this point, but it could be worth a minor version bump just to give those folks a little more of a warning?

Reply:
Yeah, good idea. The api-report method also doesn't seem to like non exported interfaces and doesn't warn about this breaking change. Maybe that is a reason to export it.

Reply:
I could even make it work without a breaking change, I'm just not sure if it's really worth in this case.

Reply:
Yeah, agree it's not worth the backward compatibility at this point, but worth exporting to make it part of the contract.

Reply:
actually, I can't get it to appear in the api report... arg!